config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root "homes#index"
+  root "monsters#index"
```

Comment:
what you should ideally be doing is pointing the root at something like homes#index, which should serve up an erb with a div that has an id of "app". From there, react with router will take over and you can have that render your monsters react components